smallrye/smallrye-mutiny
# Improve internal test utilities

This PR contains a few improvements:

﻿- The CausedTest was using the wrong @Test annotation
- Add methods to create mocks subscribers.
- Rename the MultiAssertSubscriber to AssertSubscriber as it's not a MultiSubscriber
- Add a class allowing to wrap a Subscriber into a MultiSubscriber.
- Verify the behavior of the AbstractSubscriber
